pcsx2-qt/MainWindow.cpp

```diff
@@ -977,6 +978,25 @@ bool MainWindow::shouldHideMainWindow() const
 		   QtHost::InNoGUIMode();
 }
 
+bool MainWindow::shouldAbortForMemcardBusy(const VMLock& lock)
+{
+	if (MemcardBusy::IsBusy() && !GSDumpReplayer::IsReplayingDump())
+	{
+		const QMessageBox::StandardButton res = QMessageBox::question(
+			lock.getDialogParent(), 
+			tr("WARNING: Memory Card Busy"), 
+			tr("WARNING: Your memory card is still writing data. Shutting down now will IRREVERSIBLY DESTORY YOUR MEMORY CARD. It is strongly recommended to resume your game and let it finish writing to your memory card.\n\nDo you wish to shutdown anyways and IRREVERSIBLY DESTROY YOUR MEMORY CARD?")
```

Comment:
```suggestion
tr("WARNING: Your memory card is still writing data. Shutting down now will IRREVERSIBLY DESTROY YOUR MEMORY CARD. It is strongly recommended to resume your game and let it finish writing to your memory card.\n\nDo you wish to shutdown anyways and IRREVERSIBLY DESTROY YOUR MEMORY CARD?")
````
Minor typo, but best to fix it before merging.
Aside from this looks great, nice work!